src/NUnitEngine/nunit.engine.api/TestEngineActivator.cs

```diff
@@ -39,6 +39,19 @@ public static class TestEngineActivator
         private const string DefaultAssemblyName = "nunit.engine.dll";
         internal const string DefaultTypeName = "NUnit.Engine.TestEngine";
 
+#if NETSTANDARD1_3 || NETSTANDARD2_0
+        /// <summary>
+        /// Create an instance of the test engine.
+        /// </summary>
+        /// <returns>An <see cref="NUnit.Engine.ITestEngine"/></returns>
+        public static ITestEngine CreateInstance()
+        {
+            var assemblyName = new AssemblyName(DefaultAssemblyName);
+            var assembly = Assembly.Load(assemblyName);
```

Comment:
Update - this weekend was unexpectedly sunny, so no code was written. Sorry! 🌞 

![image](https://user-images.githubusercontent.com/4837132/47272459-f9994580-d57d-11e8-9802-b439964daac2.png)

Reply:
I came across this project that works around issues with assembly loading, https://natemcmaster.com/blog/2018/07/25/netcore-plugins/ with a GitHub repo at https://github.com/natemcmaster/DotNetCorePlugins. The code is under Apache license, so we can't use it directly, but maybe if we ask nicely, the project owner might allow us to use some of his ideas in our code under MIT? I've only had a cursory look, but if it pans out, I will reach out to him.

Reply:
Gah, @rprouse - that assembly loading blog/logic is all dependent on being [AssemblyLoadContext](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.loader.assemblyloadcontext?view=netcore-2.1#applies-to) - which .NET Core rather than .NET Standard. 

I'm not sure how we can achieve this targetting .NET Standard right now. Still digging - appreciate any thoughts. 😞